site/content/en/docs/drivers/vfkit.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "vfkit"
+weight: 2
+aliases:
+    - /docs/reference/drivers/vfkit
+---
+
+## Overview
+
+[VFKit](https://github.com/crc-org/vfkit) is an open-source program for macOS virtualization, optimized for lightweight virtual machines and container deployment.
+
+## Issues
+
+### Other
+
+* [Full list of open 'vfkit' driver issues](https://github.com/kubernetes/minikube/labels/co%2Fvfkit)
+
+## Troubleshooting
+
+### Run with logs
+
+Run `minikube start --alsologtostderr -v=7` to debug crashes
```

Comment:
```suggestion
Run `minikube start --driver vfkit --alsologtostderr -v=7` to debug crashes
```
Since it's still in experimental need to explicitly pass the flag.